git-ignore.sh

```diff
@@ -77,8 +77,9 @@ get_git_ignore () {
 
 	if test $_parent_level -ne 0
 	then
-	    for i in $(seq 1 $_parent_level)
-	    do
+            i=1
+            while [ "$i" -le $_parent_level ]
+            do
```

Comment:
That misses the `i=$(($i+1))`... Without it, you run into an infinite loop.

But again, this passes on Windows, so I am even more sure that the `_parent_level` variable contains an incorrect value on Windows.

Reply:
Ahh... i accidentally erased that part and missed it... Thanks. I'll start with more diligence on the check ins :)